specifications/sep-payment-channel-transactions.md

```diff
@@ -194,29 +203,29 @@ multisig accounts. F has source account E, and sequence number set to s.
     not the native asset.
     - One `SET_OPTIONS` operation adjusting escrow account EI's thresholds such
     that I and R's signers must both sign.
-    - One or more `SET_OPTIONS` operations adding I signers to EI.
+    - One `SET_OPTIONS` operation adding I signers to EI.
```

Comment:
❓ why is this only 1 now? I can only have 1 signer?

Reply:
Multiple signers are no longer supported because we are limited to two extra signers per transactions. This limits us to two subsequent transactions, and three total transactions, during the open process. This limits us to one subsequent transaction, and two total transactions, during the update process. There is an extra unused extra signers in the update process reserved for future use as a HTLC using the hash(x) signer.

I've added notes about this in the PR description. I should have included info about this.